packages/web-components/src/global/color-mode.css

```diff
@@ -2,7 +2,6 @@
 :root,
 ic-theme,
 .ic-theme-light {
-  --ic-background-primary: var(--ic-background-primary-light);
```

Comment:
Looks like removing this has updated a badge Cypress test and removed the background of the light variant of badge